ddtrace/contrib/kombu/patch.py

```diff
@@ -30,6 +30,10 @@
 from .utils import get_routing_key_from_args
 
 
+def get_version():
+    return str(getattr(kombu, "__version__", ""))
```

Comment:
Do we need the str? kombu.__version__ should always be a string: https://github.com/celery/kombu/blob/v1.0.1/kombu/__init__.py#L3

Reply:
kombu returns unicode for Python2